src/design-system/components/Text/Text.docs.tsx

```diff
@@ -0,0 +1,145 @@
+/* eslint-disable sort-keys-fix/sort-keys-fix */
+import React, { ReactNode } from 'react';
+import { View } from 'react-native';
+import { AccentColorProvider } from '../../color/AccentColorContext';
+import { ColorModeProvider } from '../../color/ColorMode';
+import { palettes } from '../../color/palettes';
+import { CustomColor } from '../../color/useForegroundColor';
+import { Docs } from '../../playground/Docs';
+import { Guide } from '../../playground/Guide';
+import { Inline } from '../Inline/Inline';
+import { Stack } from '../Stack/Stack';
+import { Text, TextProps } from './Text';
+
+const textExamples: Required<Pick<TextProps, 'size' | 'weight'>>[] = [
+  { size: '18px', weight: 'bold' },
+  { size: '16px', weight: 'bold' },
+  { size: '14px', weight: 'bold' },
+  { size: '11px', weight: 'bold' },
+];
+
+const MockBadge = ({ children }: { children: ReactNode }) => (
+  <View style={{ display: 'flex', flexDirection: 'row' }}>
+    <View
+      style={{
+        backgroundColor: 'rgba(255,0,0,0.2)',
+        borderRadius: 999,
```

Comment:
Does this work on Android? :0

Reply:
Yep!

<img width="229" alt="Screen Shot 2021-11-24 at 8 09 28 am" src="https://user-images.githubusercontent.com/696693/143128334-85617d69-a1b4-4b6a-9f6f-556c5b16cf50.png">

Reply:
ooooo